ui/raidboss/data/triggers/orbonne_monastery.js

```diff
@@ -673,12 +673,12 @@
         'Ramza': 'Ramza',
         'Dominion': 'Dominion',
         'Demi-Famfrit': 'Demi-Famfrit',
-        '..is no longer sealed': '.*ist nicht mehr geschlossen',
-        'The Realm of the Templars will be sealed off': 'Das Reich der Tempelritter schließt',
-        'The Realm of the Machinists will be sealed off': 'Das Reich der Maschinisten schließt',
-        'The lifeless alley will be sealed off': 'Der Leblose Pfad Tempelritter schließt',
-        'The Realm of the Thunder God will be sealed off': 'Das Reich des Donnergottes schließt',
-        'The Crystalline Gaol will be sealed off': 'Der Kristallkerker schließt',
+        '..is no longer sealed': '.*öffnet sich wieder',
+        'The Realm of the Templars will be sealed off': 'Noch 15 Sekunden, bis sich der Zugang zum Reich der Tempelritter schließt',
```

Comment:
It was previously `'Das Reich der Tempelritter schließt'` and it seems like `Reich der Tempelritter schließt` would cover both sealing cases (and handle future changes to sealing time again, if they did that).  Would you mind keeping these regexes briefer and just removing Das/Der from the front rather than using the whole line.

Thanks for catching this!

Reply:
Almost! `.*Reich der Tempelritter schließt` would work i guess.